### PR TITLE
New Python shell to play around with + Docker instructions

### PR DESCRIPTION
**Pull request by map34**
* Now you can start executing Python scripts inside the server shell
* Docker instructions

### CODE REVIEW

**Comment by lsetiawan** [comment]
Here's the stack for python that I'm used to coming from the Data Science world: conda, flake8, pytest.

Should we use all standard python tools to do linting, package/environment management, and testing or those things I listed above?

File: backend/server/manage.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from oceanhub import app
+from oceanhub.scripts.shell import make_shell_context
+from flask_script import Manager, Command, Shell
```
Comment (lsetiawan) [inline backend/server/manage.py:3]: Hmm... Interesting. I haven't used flask much, didn't know about this Manager feature!

Comment (map34) [reply backend/server/manage.py:3]: yeah the manager stuff lets you bind Flask with other outside features

**Comment by map34** [comment]
@lsetiawan For testing yes, I am planning to use PyTest, for linting flake8 is perfect. For Conda, I was thinking to go lightweight at first

**Comment by lsetiawan** [comment]
> For Conda, I was thinking to go lightweight at first

@map34 Miniconda is pretty lightweight. I think it'll be good to start getting familiar with this, since it's very prominent in the Data Science community. I can help set things up.

File: backend/server/manage.py (hunk 1 above)
Comment (lsetiawan) [reply backend/server/manage.py:3]: Is this similar to Django's `manage.py`?

**Comment by map34** [comment]
@lsetiawan that would be great!

File: backend/server/manage.py (hunk 1 above)
Comment (map34) [reply backend/server/manage.py:3]: yup, pretty much

**Comment by lsetiawan** [comment]
@map34 Should we merge this first and I'll do the changes discussed above?

**Comment by map34** [comment]
let's do it!

**Pull request merged**